datajoint/relational_operand.py

```diff
@@ -229,6 +229,12 @@ def aggregate(self, group, *attributes, keep_all_rows=False, **named_attributes)
         return GroupBy.create(self, group, keep_all_rows=keep_all_rows,
                               attributes=attributes, named_attributes=named_attributes)
 
+    def aggr(self, *args, **kwargs):
```

Comment:
They belong to different classes.  The one below is for `dj.U` which does not subclass from `dj.RelationalOperand`.